VCFv4.3.tex

```diff
@@ -1362,7 +1362,7 @@ \subsection{Representing unspecified alleles and REF-only blocks (gVCF)}
 In order to report sequencing data evidence for both variant and non-variant positions in the genome, the VCF specification allows to represent blocks of reference-only calls in a single record using the END INFO tag, an idea originally introduced by the gVCF file format\footnote{\url{https://help.basespace.illumina.com/articles/descriptive/gvcf-files/}}.
 The convention adopted here is to represent reference evidence as likelihoods against an unknown alternate allele.
 Think of this as the likelihood for reference as compared to any other possible alternate allele (both SNP, indel, or otherwise).
-A symbolic alternate allele $<$*$>$ is used to represent this unspecified alternate allele.
+A symbolic alternate allele $<$*$>$ is used to represent this unspecified alternate allele. This is preferred over the symbolic allele $<$NON\_REF$>$, used by the Genome Analysis Toolkit (GATK).
```

Comment:
@cyenyxe This seems like the right solution, the community has definitely overruled my initial suggestion to promote `<NON_REF>` over `<*>`
```suggestion
A symbolic allele $<$*$>$ is used to represent this unspecified alternate allele. 
This is preferred over the equivalent allele $<$NON\_REF$>$.
```

I think we have a redundant `alternate` in there, and we should split multiple sentences on multiple lines to match the formatting.  

I think it's good to specify that they have the same meaning, and I don't know if it's necessary to include the GATK by name here since we don't usually refer to specific implementations.  

It might also be useful to highlight in some way the term `unspecified alternate allele` if that's how we intend it to be pronounced, since my major complaint with `<*>` is that it's pronunciation causes confusion.